apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -1,11 +1,12 @@
-import { protectedProcedure, router } from '~/server/trpc'
+import { schema, type IsomerPageSchema } from '@opengovsg/isomer-components'
```

Comment:
schema should now be exported from the isomer-components package right? In this PR I still see it within docs folder

Reply:
Yup, there are some issues with using symbolic links in GitHub Pages. Not doing that right now because [that requires us to convert the GitHub Pages build process into GitHub Actions](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#troubleshooting-publishing-from-a-branch) so that they can support symbolic links.

Reply:
Why do we need symbolic links? Instead of the schema being in the docs, we just move it to the components package and serve that right?

Reply:
Yeah but the `docs` folder is a special one that GitHub Pages uses to build the site, so the file has to be present in that folder for it to appear in schema.isomer.gov.sg.

We are keeping that schema because that's being used right now by POEs for their migration work, since Studio is not available for them to use yet.